index.html

```diff
@@ -178,7 +178,6 @@ <h3 class="ftr">I'm always interested in hearing about new projects, so if you'd
             </ul>
         </footer>
     </main>
```

Comment:
- [ ] Please make sure to update the main page so that the projects section is created dynamically using the information stored in that JavaScript array. All of the HTML in that section should be created when the page loads as requested in the project requirements.

Reply:
What do you exactly mean by this issue? Because every html is being creeated only when the user presses the button for popup view of the project. Check it again please. All the popup html elements are being created in popup.js file.

Reply:
@Ahsan12356 the entire project section in the index.html should be created using javascript